sbi/utils/metrics.py

```diff
@@ -20,22 +20,57 @@ def c2st(
     scoring: str = "accuracy",
     z_score: bool = True,
     noise_scale: Optional[float] = None,
+    verbosity: int = 0,
+    clf_class=RandomForestClassifier,
+    clf_kwargs={},
 ) -> Tensor:
-    """Return accuracy of classifier trained to distinguish samples from two distributions.
+    """
+    Return accuracy of classifier trained to distinguish samples from supposedly
+    two distributions <X> and <Y>. For details on the method, see [1,2].
+    If the returned accuracy is 0.5, <X> and <Y> are considered to be from the
+    same generating PDF, i.e. they can not be differentiated.
+    If the returned accuracy is around 1., <X> and <Y> are considered to be from
+    two different generating PDFs.
+
+    Training of the classifier with N-fold cross-validation [3] using sklearn.
+    By default, a `RandomForestClassifier` by from `sklearn.ensemble` is used.
```

Comment:
change to sth like: the current implementation allows to use either a MLP classifier or a RandomForestClassifier as implemented in sklearn. We recommend using X.

Reply:
We could resort to this `str` parameter that you suggest, but I fear that it makes the interface more rigid than it needs to be. What if at some point, `sbi` becomes able to use images as input? Then you want to interchange the classifyer to a `resnet` etc.

What I could foresee is, that we wrap the more detailed `c2st` implementation which yields the `str` argument @janfb proposed. This way, people can use a more flexible interface or a very simple one.

Reply:
I see, you mean that users could pass their own classifiers which they want to use for running c2st?

Reply:
Yes, this is what this implementation of `c2st` does already support.

Reply:
I hope I honored these requests with 5fd41c5